README.md

```diff
@@ -1,21 +1,36 @@
-# Zarf - Kubernetes Air Gap Buddy 
+# Zarf - DevSecOps for Air Gap Systems 
 
 <img align="right" alt="zarf logo" src=".images/zarf-logo.png"  height="256" />
 
-Zarf massively simplifies the setup & administration of kubernetes clusters "across the [air gap](https://en.wikipedia.org/wiki/Air_gap_(networking))".
+## Why is Zarf Needed In This World?
+
+Most of the software ecosystem assumes your systems have access to the internet.  The world (for good reasons) has become more and more dependent upon Software as a Serivice (SaaS), which assume a robust connection to the internet and a willingness to inheritently trust 3rd party providers. Although this makes sense for 99% of the world, there are certain SECURE systems that must operate either fully disconencted, semi-disconnected, or might just want the ability to disconnect in case of emergencies (like while under an active cyber attack). Although only a small percentage of systems, these SECURE systems make up some of the most vital systems in the world, such as Aerospace and Defense, Finance, Healthcare, Energy, Water and Sewage, as well as many Federal, Local, and State Goverment systems.  
+
+These SECURE systems need a way to continously and securely deliver software too. Zarf exists to make it easy for open source, COTS, and organic software solutions to be delivered to secure and disconnected systems. Because although such systems might be small in number, they represent many of the most important systems in the world.  
```

Comment:
> make it easy for open source, COTS...

Open Source is almost always still considered COTS. See [DoD Open Source FAQ](https://dodcio.defense.gov/open-source-software-faq/#q-is-oss-commercial-software-is-it-cots). Recommend changing to "open source, proprietary, and organic software solutions". This is an important distinction because it helps the large parts of the DoD that are still skeptical of open source to understand that "commercial software" can, and usually does, include open source.